### PR TITLE
Tweak workflows

### PR DESCRIPTION
**Pull request by ndilalla**

### CODE REVIEW

File: .github/workflows/python-package-conda.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: Python Package using Conda
 on:
   #push:
   schedule:
-    - cron: "0 20 * * *"
```
Comment (henrikef) [inline .github/workflows/python-package-conda.yml:6]: Hm, just curious: Why did you change this one to run monthly?

**Review by henrikef — APPROVED** [review]

File: .github/workflows/python-package-conda.yml (hunk 1 above)
Comment (ndilalla) [reply .github/workflows/python-package-conda.yml:6]: Because it is failing every day and I wanted to disable it until I merge the new version I developed in a separate branch.

Comment (henrikef) [reply .github/workflows/python-package-conda.yml:6]: Ah, makes sense.

**Comment by ndilalla** [comment]
> Hm, more issues with the fermi data download. Maybe it's worth setting the fermipy tests to xfail again for now?

Yes, I think it's the right thing to do. I'll do it now.
